### PR TITLE
fix: cleanup onboarding optimization experiment

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Removal of animation experiment (result was set to true (so no animations etc anymore))

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### Preview domain
https://as-206-cleanup-onboarding-exp.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/filters/MyFeedHeading.tsx [hunk 1]
```diff
@@ -73,7 +61,7 @@ function MyFeedHeading({
 
   const alertProps: Omit<AlertPointerProps, 'children'> = {
     offset: getOffset(),
-    isAlertDisabled: shouldHighlightFeedSettings ? isAlertDisabled : true,
+    isAlertDisabled: true,
```
Comment (sshanzel) [inline packages/shared/src/components/filters/MyFeedHeading.tsx:64]: This means there will never be an alert, should we remove it completely instead?

**Review by sshanzel — APPROVED** [review]
I always love cleanups. Just had one comment about something we can maybe cleanup as well.

File: packages/shared/src/components/filters/MyFeedHeading.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/filters/MyFeedHeading.tsx:64]: Ah yes, wasn't sure what the alert was here, but can check and do that yes.

**Review by nensidosari — APPROVED** [review]
Love it, let's remove from [growthbook](https://app.growthbook.io/features/onboarding_optimizations) as well after this PR is merged 🙏

**Review by nensidosari — APPROVED** [review]

**Pull request merged**